Command/EventLogCleanupCommand.php

```diff
@@ -67,8 +68,11 @@ protected function configure(): void
                 Purge only lead_event_log records
                 <info>php %command.full_name% --lead</info> 
                 
-                Purge only email_stats and email_stats_devices records:
-                <info>php %command.full_name% --email-stats </info> 
+                Purge only email_stats the referenced emails entry is currently not published and email_stats_devices records [Important: If referenced email is ever switched back to published, the contacts will get the email again]:
```

Comment:
Correction: "Purge only email_stats **where** the referenced email~~s~~ entry..."

Reply:
Thanks @moongazer for this spelling recommendation - implemented that with my current commit.